dspace/config/registries/schema-person-types.xml

```diff
@@ -64,5 +72,63 @@
         <element>jobTitle</element>
         <scope_note>The job title of the person (for example, Financial Manager).</scope_note>
     </dc-type>
+	
+    <!--
+      OpenAIRE4 Guidelines
+      2.2.6. Subproperty affiliation (R)
+      The organizational or institutional affiliation of the creator (occurrence: 0-n). -->
+    <dc-type>
+        <schema>person</schema>
+        <element>affiliation</element>
+        <qualifier>name</qualifier>
+        <scope_note>The organizational or institutional affiliation of the creator</scope_note>
+    </dc-type>
+    
+    <!-- OpenAIRE4 Guidelines -->
+    <!-- 2.2.2.1. Attribute nameType (R) -->
+    <!-- and 3.2.6. Subproperty nameIdentifier (R) -->
+    <!-- specific type required by OpenAIRE4 Guidelines -->  
+
+    <dc-type>
+        <schema>person</schema>
+        <element>identifier</element>
+        <qualifier>authorid</qualifier>
+        <scope_note>Scopus Author ID</scope_note>
```

Comment:
I would prefer to call this `person.identifier.scopusid`.   Calling it "authorid" is a bit too generic of a term, as there are many different types of author identifiers.

Reply:
I agree the name is generic and I don't even remember where we saw it. Unlike Researcher ID that mentions that it can be used a RID. ORCID just refers to a `"Scopus Author ID"` (https://support.orcid.org/hc/en-us/articles/360006894854-Add-person-identifiers-other-identifiers-to-your-ORCID-record) 
`<common:external-id-type>Scopus Author ID</common:external-id-type>`. I propose to in some way follow what ORCID does and instead use `scopus-author-id`. I don't know if `scopus-id` may be used in the context of  publications.

Reply:
I also agree that the name should be more specific, since the meaning expressed in the scope_note certainly is.  But I wonder:  aren't these standardized somewhere?  If we just make up field names, how will other sites understand them?

Reply:
@paulo-graca : I'm OK with renaming this qualifier to be `scopus-author-id`.  That's fine by me.

Reply:
Thank you @mwoodiupui ! We share this same concern. When we initially did this PR we tried to use standardized names as much as possible. That is the case, for instance of ResearcherID which can be also used `RID`.  The Scopus **authorid** case it's only used in Scopus internal context. Outside, I totally agree that this name it's a bit generic. 
ORCID, for instance, refers to it just as `<common:external-id-type>Scopus Author ID</common:external-id-type>` . ORCID also exposes this list:
https://pub.orcid.org/v2.0/identifiers, but it seams to only refer to Works. I've no better suggestion for this and whatever better suggestions are welcome!